### DISCUSSION TITLE
Fabric version?

### DISCUSSION THREAD

**Discussion by Vandomas** [P3pp3rF1y/SophisticatedBackpacks#998] [Q&A] [answered]
Do you have plans to implement this wonderful mod in fabric?

**Comment by P3pp3rF1y** [comment, marked as answer]
There's this now https://www.curseforge.com/minecraft/mc-mods/sophisticated-backpacks-unofficial-fabric-port . I may do an official one in the future, but until then this is what you can use.

**Reply by Vandomas** [reply]
Thank you very match! I use this port, but very waiting for your official version!
